streaming/multimodal/webvid.py

```diff
@@ -40,3 +44,195 @@ def __getitem__(self, idx: int) -> Any:
         obj = super().__getitem__(idx)
         # Processing goes here.
         return obj
+
+
+class StreamingOutsideGIWebVid(StreamingDataset):
+    """Streaming WebVid dataset.
```

Comment:
Can you please update the description on what Streaming Outside GI WebVid means? Sorry I have little context what each of `outside`, `inside`, `gi`, `dt` means!